src/theme/index.ts

```diff
@@ -47,6 +47,7 @@ import CustomSlider from "./components/slider";
 import StatusBadge from "./components/statusBadge";
```

Comment:
Take a look at the `src/index.ts` file. The component and the `SocialMediaLinkDataProps` type should be imported and exported in that file so that it's accessible when the project is released on npm. You'll see the syntax for the component and the type in that file as well.